daemons/gptp/common/ptp_message.cpp

```diff
@@ -959,9 +962,8 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 	master_local_freq_offset += 1.0;
```

Comment:
Thanks for catching that. We were so engrossed in making sure correctionField value is added correctly we missed the values being used in the frequency calculation. Do you want me to make this change and add it to this pull request? 

Also a side note I just noticed that we send the pull request to the master branch. Do you want me to re initiate the pull request to the open-avb-next branch instead?